packages/elements/src/internals/machines/sign-in/types/router.types.ts

```diff
@@ -93,3 +93,18 @@ export interface SignInRouterSchema {
   events: SignInRouterEvents;
   tags: SignInRouterTags;
 }
+
+// ---------------------------------- Schema ---------------------------------- //
+
+export type SignInChildren = any; // TODO: Update
+export type SignInOuptut = any; // TODO: Update
+export type SignInStateValue = any; // TODO: Update
+
+export type SignInRouterSnapshot = MachineSnapshot<
+  SignInRouterContext,
+  SignInRouterEvents,
+  SignInChildren,
+  SignInStateValue,
+  SignInRouterTags,
+  SignInOuptut
+>;
```

Comment:
What's this change for? 👀

Reply:
A central place to retain the type for a SignInRouter snapshot... as used largely by `useSelector`.